src/workflows-app/SubmissionDetails.js

```diff
@@ -230,7 +230,7 @@ export const BaseSubmissionDetails = ({ name, namespace, workspace, submissionId
     const breadcrumbPathObjects = [
       {
         label: 'Submission History',
-        path: 'workspace-workflows-app-submission-history',
+        path: 'workspace-workflows-app', // TODO: Refactor to reroute to submission history tab in panel
```

Comment:
is there a ticket for this already? if so, might be useful to point to it in this comment!